who_is_active.sql

```diff
@@ -3108,6 +3111,7 @@ BEGIN;
                         COALESCE(r.ansi_nulls, s.ansi_nulls) AS ansi_nulls,
                         COALESCE(r.concat_null_yields_null, s.concat_null_yields_null) AS concat_null_yields_null,
                         COALESCE(r.transaction_isolation_level, s.transaction_isolation_level) AS transaction_isolation_level,
+                        c.encrypt_option,

```

Comment:
Should we do the following instead? 
```suggestion
                        COALESCE(c.encrypt_option, 'FALSE') as encrypt_option,

```